packages/features/ee/teams/lib/payments.ts

```diff
@@ -106,6 +108,17 @@ export const purchaseTeamOrOrgSubscription = async (input: {
 
   const customer = await getStripeCustomerIdFromUserId(userId);
 
+  const fixedPrice = await getFixedPrice();
+
+  const customPrice = await createPrice({
```

Comment:
Price creation has to be conditional, in self-serve he would be using fixedPrice only.

I think this was done based on pricePerSeat variable